File: CursoPOOUber/PHP/UberBlack.php
<?php
require_once('car.php');
class UberBlack extends Car{
 public $typeCarAccepted;
 public $seatmaterial;

 public function __constructor($license, $driver
        $typeCarAccepted,$seatmaterial){
            parent::__constructor($license,$driver);
            $this->typeCareAccepted =$typeCarAccepted;
            $this->seatmaterial = $seatmaterial;
    }
}

?>

File: CursoPOOUber/Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver,typeCarAccept, seatsMaterial):
        super.__init__(license, driver)
        self.typeCarAccepted = typeCarAccept
        self.seatsMaterial = seatsMaterial
